mobile/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,5 @@
     <color name="colorPrimary">#3F51B5</color>
     <color name="colorPrimaryDark">#303F9F</color>
     <color name="colorAccent">#FF4081</color>
+    <color name="orange">#AE5733</color>

```

Comment:
Nevermind I see this now. I don't know what the other thing was.

Reply:
it was the color we decided when we went with green